src/main/java/org/toradocu/generator/OracleGenerator.java

```diff
@@ -44,11 +45,14 @@ public static void createAspects(List<DocumentedMethod> methods) {
     }
 
     String aspectDir = Toradocu.configuration.getAspectsOutputDir();
-    try {
-      Files.createDirectory(Paths.get(aspectDir));
-    } catch (IOException e) {
-      log.error("Error while creating" + aspectDir + ".", e);
-      System.exit(1);
+    Path aspectDirPath = Paths.get(aspectDir);
+    if (!Files.exists(aspectDirPath)) {
+      try {
+        Files.createDirectory(aspectDirPath);
```

Comment:
I think it's be better to keep the check on the existence of the aspectDirPath and the creation as an atomic instruction. I would rewrite the code as
```java
Path aspectDirPath = Paths.get(aspectDir);
try {
  Files.createDirectory(aspectDirPath);
} catch (FileAlreadyExistsException e) {
} catch (IOException e) {
  log.error("Error while creating directory \"" + aspectDir + "\".", e);
  System.exit(1);
}
```

Do you agree?